restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentSearchPageState.java

```diff
@@ -1,65 +0,0 @@
-package io.stargate.web.docsapi.dao;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import java.nio.ByteBuffer;
-import java.util.Base64;
-
-public class DocumentSearchPageState {
```

Comment:
This class was not used. I noticed it now because it used `Base64` (incorrectly).